lib/renderJs.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+'use strict';
+
+const SassValue = require('./SassValue');
+
+function createJsonMap(data) {
+  return JSON.stringify(
+    data,
+    (key, value) =>
+      value instanceof SassValue
+        ? value.fallback
+          ? value.fallback
+          : undefined
+        : value,
+    2
+  );
+}
+
+function renderJs(data) {
+  let output = '';
+  if (data.gesso) {
+    const gessoCategories = Object.keys(data.gesso);
```

Comment:
Why not use [`Object.entries()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) here?

Reply:
Because I remembered `Object.keys` but not `Object.entries()`. 😛 Will fix.